apps/aecore/src/aec_chain_server.erl

```diff
@@ -107,6 +107,26 @@ handle_call(difficulty, _From, State) ->
 handle_call({common_ancestor, Hash1, Hash2}, _From, State) ->
     {reply,
      aec_chain_state:find_common_ancestor(Hash1, Hash2,State), State};
+handle_call(get_genesis_header, _From, State) ->
+    GenesisHash = aec_chain_state:get_genesis_hash(State),
+    Resp =
+        case aec_chain_state:get_header(GenesisHash, State) of
+            error ->
```

Comment:
Is not the presence of the genesis guaranteed by the `init` and by the persisted chain not being corrupted?
If so, I expect this case to be replaced with hardcoded expectation (match) that the genesis_hash is in state.